frontend/src/lib/constants.tsx

```diff
@@ -232,6 +232,7 @@ export const FEATURE_FLAGS = {
     CUSTOM_CHANNEL_TYPE_RULES: 'custom-channel-type-rules', // owner: @robbie-c #team-web-analytics
     SELF_SERVE_CREDIT_OVERRIDE: 'self-serve-credit-override', // owner: @zach
     EXPERIMENTS_MIGRATION_DISABLE_UI: 'experiments-migration-disable-ui', // owner: @jurajmajerik #team-experiments
+    METALYTICS: 'metalytics', // owner: @surbhi
```

Comment:
Don't forget to add this to prod when shipping.

Reply:
Do i need to do anything additional to this to make sure its in prod?